### PR TITLE
Add data stream type "traces".

### LINKED ISSUE #68: Create traces data type
The spec defines 2 data types: metrics and logs (https://github.com/elastic/package-spec/blob/e7102f8cbd392d977bb2a21b12674dbe2ae8b985/versions/1/data_stream/manifest.spec.yml#L93-L95).

For the upcoming APM integration we need a new `traces` data type.

### PR DESCRIPTION
**Pull request by jalvz**
Closes #68

### CODE REVIEW

**Review by ruflin — APPROVED** [review]
LGTM.

@ycombinator Could you check if this is the only place it needs to be added?

**Comment by jalvz** [comment]
@ycombinator does this looks good to you?

File: versions/1/changelog.yml [hunk 1]
```diff
@@ -5,10 +5,13 @@
   changes:
   - description: Initial version of the Elastic Package specifiction.
     type: enhancement
-    link: https://github.com/elastic/package-spec/pulls/1
+    link: https://github.com/elastic/package-spec/pull/1
```
Comment (ycombinator) [inline versions/1/changelog.yml:8]: Thanks for fixing this. <3

**Review by ycombinator — APPROVED** [review]
There's a merge conflict. But changes LGTM.

**Pull request merged**